pyproject.toml

```diff
@@ -251,31 +249,15 @@ tests = [
 
 # CI related soft dependency sets - not for users of sktime, only for developers
 # these are for specual uses and may be changed or removed at any time
-binder = [
-  "jupyter",
-  "pandas<2.0.0",
-]
-cython_extras = [
-  "mrseql",
-  'mrsqm; python_version < "3.11"',
-  "numba<0.59",
-]
+binder = ["jupyter", "pandas<2.0.0"]
```

Comment:
please do not do that - internal testing code expects all dependencies to be on a new line (for parsing)

Reply:
didn't check the diff before submitting the PR, my bad, these changes were all made by format on save - will reverse them